Packs/Akamai_WAF/ReleaseNotes/2_0_3.md

```diff
@@ -0,0 +1,19 @@
+
+#### Integrations
+##### Akamai WAF
+- Adding one new command and modifying two commands.
```

Comment:
Rewrite the RN as follows:
- Added the ***acknowledge_pre_verification_warning*** command.  When there is a conflict between the existing certificate and the new certificate, a warning message will appear, which needs to be acknowledged either manually or automatically by an API call. This command acknowledges the warning. After it is acknowledged, the status of the enrollment will move to the next phase.
- Changed the read timeout value from 60 to 180 in the ***list_enrollments*** command to avoid a potential timeout issue.
- Added the *rule_update* parameter to the clone_appsec_config_version command. When you set *rule_update* to "False" the command execution will take less than 60 seconds because it eliminates the rule schema update. For any Akamai deployment with "ASE" enabled for the Security configuration, the rule schema is updated automatically anyway.